apps/evm/src/pages/Market/Market/MarketHistory/Card/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { useMemo, useState } from 'react';
+
+import { ButtonGroup, Spinner } from 'components';
+import { ApyChart, type ApyChartProps } from 'components/charts/ApyChart';
+import { useTranslation } from 'libs/translations';
+import type { Asset } from 'types';
+import {
+  formatCentsToReadableValue,
+  formatPercentageToReadableValue,
+  getCombinedDistributionApys,
+} from 'utilities';
+
+import { useGetPoolLiquidationIncentive } from 'clients/api';
+import { useIsFeatureEnabled } from 'hooks/useIsFeatureEnabled';
+import { MarketCard, type MarketCardProps } from '../../MarketCard';
+import { useGetLiquidationThresholdPercentage } from './useGetLiquidationThresholdPercentage';
+
+const ENTRIES_PER_DAY = 1;
+const ENTRIES_PER_30_DAYS = ENTRIES_PER_DAY * 30;
+const ENTRIES_PER_6_MONTHS = ENTRIES_PER_30_DAYS * 6;
+const ENTRIES_PER_YEAR = ENTRIES_PER_6_MONTHS * 2;
+
+export interface CardProps {
+  type: ApyChartProps['type'];
+  asset: Asset;
+  data: ApyChartProps['data'];
+  poolComptrollerContractAddress: string;
+  isLoading: boolean;
+  testId: string;
+}
+
+export const Card: React.FC<CardProps> = ({
+  type,
+  data,
+  isLoading,
+  asset,
+  poolComptrollerContractAddress,
+  ...otherProps
+}) => {
+  const { t } = useTranslation();
+  const isApyChartsFeatureEnabled = useIsFeatureEnabled({ name: 'apyCharts' });
+  const isNewMarketPageEnabled = useIsFeatureEnabled({ name: 'newMarketPage' });
+  const shouldDisplayLiquidationInfo = isNewMarketPageEnabled && type === 'borrow';
+
+  const [selectedPeriodIndex, setSelectedPeriodIndex] = useState(2);
+  const periodOptions = useMemo(
+    () => [
+      {
+        label: t('market.periodOption.oneYear'),
+        value: ENTRIES_PER_YEAR,
+      },
+      {
+        label: t('market.periodOption.sixMonths'),
+        value: ENTRIES_PER_6_MONTHS,
+      },
+      {
+        label: t('market.periodOption.thirtyDays'),
+        value: ENTRIES_PER_30_DAYS,
+      },
+    ],
+    [t],
+  );
```

Comment:
Very nitpicky comment, but I feel it's more natural to have the leftmost option be 30 days and the rightmost option be 1 year

Reply:
Ok, let's do that!